scripts/release-tasks.ts

```diff
@@ -1,7 +1,7 @@
 import color from 'ansi-colors';
 import Listr, { ListrTask } from 'listr';
 
-import { bundleBuild } from './build';
+import { buildAll } from './esbuild/build';
```

Comment:
I _think_ this is the only change we need to make to the existing release scripts - just swapping out a function that runs all of the esbuild bundles for the old one that runs the rollup bundles.